scripts/Tools/create_train

```diff
@@ -1,244 +0,0 @@
-#!/usr/bin/env perl
```

Comment:
This script is completely broken - the only reason to support trains is to satisfy the needs of some HPC centers that want you to run very large jobs.   I submit that we should remove this and add a new script if we at some future time have a need to support trains.